modules/apps/27-interchain-accounts/keeper/account.go

```diff
@@ -56,3 +56,16 @@ func (k Keeper) RegisterInterchainAccount(ctx sdk.Context, accAddr sdk.AccAddres
 	k.accountKeeper.SetAccount(ctx, interchainAccount)
 	k.SetInterchainAccountAddress(ctx, controllerPortID, interchainAccount.Address)
 }
+
+// Opens a new channel on a particular port given a connection
+// This is a helper function to open a new channel
+// in case of a channel closing and the controller chain needs to regain access to an interchain account on the host chain
+func (k Keeper) InitChannel(ctx sdk.Context, portID, connectionID string) error {
```

Comment:
Not sure about the naming here. I would like some opinions on this.

Reply:
ReopenChannel? I'm not sure either, but I wouldn't want external users to start using `InitChannel` in place of `InitInterchainAccount`

Reply:
Yeah, good point.